CMakeLists.txt

```diff
@@ -535,10 +535,10 @@ endif()
 
 if(${use_mbedtls})
     if (WIN32)
-        set(aziotsharedutil_target_libs ${aziotsharedutil_target_libs} 
-            $ENV{mbedtlsLibDir}/mbedtls 
-            $ENV{mbedtlsLibDir}/mbedx509 
-            $ENV{mbedtlsLibDir}/mbedcrypto
+        set(aziotsharedutil_target_libs ${aziotsharedutil_target_libs} ws2_32
```

Comment:
what is `ws2_32`?

Reply:
Windows socket library.
https://docs.microsoft.com/en-us/windows/win32/winsock/creating-a-basic-winsock-application